README.md

```diff
@@ -1,4 +1,6 @@
-Statsbook Tool is a utility for processing WFTDA statsbook files.  It will read a WFTDA Excel spreadsheet file and produce a list of detected errors.  It will also save the game in DerbyJSON format, in case that proves useful. This software is not provided, endorsed, produced, or supported by the WFTDA.  It is not guaranteed to work, to be free of false positives, false negatives, random bugs, or apocalyptic curses.
+Statsbook Tool is a utility for processing WFTDA statsbook files.  It will read a WFTDA Excel spreadsheet file and produce a list of detected errors.  In addition, it can export rosters from the IGRF page of the StatsBook in .xml format for importation to CRG Scoreboard. It can also save the game in DerbyJSON format, in case that proves useful for some reason. 
+
+This software is not provided, endorsed, produced, or supported by the WFTDA.  It is not guaranteed to work, to be free of false positives, false negatives, random bugs, or apocalyptic curses.
```

Comment:
I have spent over an hour wrestling with git and github to try to just submit the changes I want to make, but it insists on showing changes you've made since my last pull request as if they were MY changes. I have no idea of how to fix that, but, since these are already your changes, hopefully that's ok?